.circleci/config.yml

```diff
@@ -1,7 +1,12 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@2.2
+  hmpps: ministryofjustice/hmpps@10.1.1
+
+executors:
+  default:
+    docker:
+      - image: circleci/python:3.8
```

Comment:
just curious what python here is being used for? do we need the default executor?